.github/stale.yml

```diff
@@ -13,9 +13,20 @@ exemptLabels:
 staleLabel: stale
 # Comment to post when marking as stale. Set to `false` to disable
 markComment: >
+  Thanks for your contribution!
+
   This issue has been automatically marked as stale because it has not had
-  recent activity. It will be closed if no further activity occurs. Thank you
-  for your contributions.
+  recent activity. Because the Atom team treats their issues
+  [as their backlog](https://en.wikipedia.org/wiki/Scrum_(software_development)#Product_backlog) stale issues
+  are closed. If you would like this issue to remain open:
+
+  1. Verify that you can still reproduce the issue in the latest version of Atom
+  1. Comment that the issue is still reproducible and include:
+    * What version of Atom you reproduced the issue on
+    * What OS and version you reproduced the issue on
+    * What steps you followed to reproduce the issue
+
+  Issues that are labeled as triaged will not be automatically marked as stale.
```

Comment:
Is this necessary?

Reply:
Not necessary, though @Ben3eeE felt that it would be helpful because it is a common question.

Reply:
Yeah there was some confusion around this. Like "This is labeled as bug does that mean...". I don't think it hurts to be explicit about it so that users will know. We do also not automatically close security and regression labeled issues but I intentionally left those out because then it was too complicated and getting your issue labeled as triaged seems like the most common thing to prevent the bot from automatically closing it.